### ISSUE TITLE
Error on POST to API /api/statuses/update

### ISSUE THREAD

**Issue by Flashwalker** [redmatrix/hubzilla#808]
Is it `/api/statuses/update` still supported or droped?

Just tested on few hubs with versions 2.4, 2.5:  
**POST** to:  
`curl -ksSi -u "$login":"$password" -X POST -d "title=bla-bla&category=bla-bla&channel=" --data-urlencode "htmlstatus=bla-bla" https://hub.vilarejo.pro.br/api/statuses/update`
and got:  
**404 Not Found**  
`{"error":"not implemented"}`
But **GET** request:  
`curl -ksSi -u "$login":"$password" -X GET https://hub.vilarejo.pro.br/api/statuses/update`  
is **200 OK**

But **POST** still works **200 OK** on version 2.2 (hub.feder8.ru hub).

Also, on lastauth.com hub after update (version 2.4 or higher) it is:  

with **GET** requests to:  
`/api/account/verify_credentials`  
`/api/statuses/update`  
basic auth works and it's **200 OK**

but with **POST** to:  
`/api/statuses/update`  
basic auth works and it's **500 Internal Server Error**

**Comment by zotlabs** [comment]
That particular function is part of the Twitter/StatusNet compatible API. This is now provided in an addon and enabling it is a site decision. It is no longer installed by default. 

By default Hubzilla provides the 'Zot API' which is described in your site documentation. 

For creating a status post you may want to look at 
[siteurl]/help/developer/api_zot#item_update

The 500 error appears to be an issue with lastauth.com - I could not reliably reproduce it, though I've seen it before on shared hosts under heavy delivery loads. Optimising the delivery settings often takes a bit of fiddling.

**Comment by Flashwalker** [comment]
Thanks

**Comment by zotlabs** [comment]
I've tried disabling the Twitter_API plugin to try and determine why you get 200 OK on a GET request to one of the now disabled plugin functions. I cannot duplicate that. If the plugin is disabled I always get a 404 on api/statuses/update with either GET or POST. So this is again likely to be a configuration issue but I'm not certain how to trigger it. (I've checked and vilarejo does not have the twitter_api plugin installed. Lastauth does so it's OK to have GET api/statuses/update return 200 there).
